ansible/ansible.cfg

```diff
@@ -7,3 +7,5 @@ forks = 20
 
 [ssh_connection]
 ssh_args = -C -o ControlMaster=auto -o ControlPersist=60s -o ServerAliveInterval=30 -o ServerAliveCountMax=10
+#ssh_args =
+scp_if_ssh=True
```

Comment:
what is this variable for?

Reply:
for the SSH issue debugging purpose this flag was suggested from askubuntu.com 
lets see if that works without this flag